homework/9.js

```diff
@@ -0,0 +1,21 @@
+/*9. If x equals 7, and the only other statement is x = x % 3, 
+what would be the new value of x?*/
+ let x = 7;
+ x = x % 3;
+ console.log(x);
+
+
+/*9.1 Add at least 3 console.log statements 
+in which you show that you understand what % does.*/
+
+let y = 10;
+ y = y % 2;
+ console.log(y);
+
+ let z = 15;
+ k = z % 4;
+ console.log(k);
+
+ let iAmRunningOutOfLetters = 17;
+ k = iAmRunningOutOfLetters % 11;
+ console.log(k);
```

Comment:
In the first example you use the same variable for assignment and operand (y). JavaScript is lenient ('forgiving') so no problem but it is good to be aware of this. What could (theoretically) be a problem? I usually don't do this but do like your othre examples z and k, iAmRunningOutOfLetters and k.